sr_port/hd.mpt

```diff
@@ -11,13 +14,17 @@
 %HD	;GT.M %HD utility - hexadecimal to decimal conversion program
 	;invoke at INT with %HD in hexadecimal to return %HD in decimal
 	;invoke at FUNC as an extrinsic function
+	;Note: works even if input hex number contains 0x or 0X prefix
```

Comment:
``Note: Ignores any prefix of 0x or 0X``

Reply:
Addressed in next commit. Please review that.